ui/lib/ldap/addon/routes/error.ts

```diff
@@ -7,7 +7,7 @@ import Route from '@ember/routing/route';
 import { service } from '@ember/service';
 
 import type Transition from '@ember/routing/transition';
-import type AdapterError from 'ember-data/adapter'; // eslint-disable-line ember/use-ember-data-rfc-395-imports
+import type AdapterError from '@ember-data/adapter/error';
```

Comment:
Really loving the consistency now! Historically I had trouble discerning where I should be importing this from 🤩